requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Versions are pinned to prevent pypi releases arbitrarily breaking
 # tests with new APIs/semantics. We want to update versions deliberately.
 ansible==2.2.2.0
-boto==2.45.0
+boto==2.34.0
```

Comment:
It was intentional... I tried to explain it (briefly) in the PR description: I initially picked 2.45.0 just because it was what I had in the env where I was smoke-testing this. However, 2.34 is the version that is used in RHEL7 / will be used in the downstream image, which means it should be a better candidate version than "what Pep had around" ;)

I'm not aware of a policy/way to pick and maintain the versions pinned here, so if there's a better approach please let me know.